### DISCUSSION TITLE
苹果m1适配问题

### DISCUSSION THREAD

**Discussion by bingling00** [tiddly-gittly/TidGi-Desktop#90] [Q&A] [answered]
1、0.4.1能打开，但不能用。创建本地wiki，可以创建，侧边栏 显示了图标，但并不能打开。主界面一直显示加载中。
2、创建时 登陆github，链接后，创建界面 变白。
    2.1 关闭后，再次点击“克隆线上wiki”，依旧会变白。
    2.2 填写信息后，点击创建，进度条会一直显示滚动。
3、暗色皮肤适配查，字体依旧是黑色，很难辨认。

**Comment by bingling00** [comment, marked as answer]
我重新新建了笔记，当只有一个本地笔记时，重启可以使用。

**Reply by linonetwo** [reply]
其实出现「加载中」的时候，可以 cmd+R 刷新一下，然后点击出现的「reload」按钮。好像有时候启动的时候加载慢了会被以为加载失败。

**Reply by linonetwo** [reply]
2 没看懂，来个截图看看，是不是和 https://github.com/tiddly-gittly/TiddlyGit-Desktop/issues/88 一样？

**Reply by bingling00** [reply]
> 2 没看懂，来个截图看看，是不是和 #88 一样？
好像不太一样，你看吧，我不懂
![image](https://user-images.githubusercontent.com/61322616/128357322-0edfb3e2-1637-40f7-842a-73b7b4ad853f.png)

**Reply by linonetwo** [reply]
那好像是一样的问题，你这是 windows 吧，我 Mac 上是好的。

**Reply by linonetwo** [reply]
可以看看 https://github.com/tiddly-gittly/TiddlyGit-Desktop/releases/tag/v0.4.2 是不是修复了

**Comment by bingling00** [comment]
我重新新建了笔记，当只有一个本地笔记时，重启可以使用。
